client/me/security-ssh-key/manage-ssh-keys.tsx

```diff
@@ -25,6 +32,17 @@ const SSHKey = ( { userLogin, sshKey, onDelete, keyBeingDeleted }: SSHKeyProps )
 		);
 	};
 
+	const handleUpdateClick = () => {
+		accept(
```

Comment:
Instead of having two modals, which is a little bit of an odd experience, let's simply add a 'Cancel' button to the modal with the form:

<img width="804" alt="image" src="https://user-images.githubusercontent.com/36432/201770789-a4fb7090-1116-48bc-8133-cddc5d2e1fce.png">

One modal would be much better.